src/components/application_manager/src/rpc_service_impl.cc

```diff
@@ -535,13 +527,58 @@ void RPCServiceImpl::SendMessageToMobile(
                (*message)[jhs::S_PARAMS][jhs::S_MESSAGE_TYPE].asInt())) {
     allow_unknown_parameters = false;
   }
+
+  const auto api_function_id = static_cast<mobile_apis::FunctionID::eType>(
```

Comment:
@YarikMamykin  do we need api_function_id outside the if?

Reply:
@LuxoftAKutsan yes, it is used in code below.